odh-6562 update certificates section (#693)
opendatahub-io/opendatahub-documentation: modules/adding-certificates-to-a-cluster-ca-bundle.adoc

:_module-type: PROCEDURE

[id="adding-certificates-to-a-cluster-ca-bundle_{context}"]
= Adding certificates to a cluster-wide CA bundle

[role='_abstract']

> You can add a self-signed certificate to a cluster-wide Certificate Authority (CA) bundle (`ca-bundle.crt``).  

Reviewer: ```suggestion
You can add a self-signed certificate to a cluster-wide Certificate Authority (CA) bundle (`ca-bundle.crt`).  
```

endif::[]

ifdef::self-managed[]
ifndef::disconnected[]
Alternatively, you can add certificates to a custom CA bundle, as described in link:{rhoaidocshome}{default-format-url}/installing_and_uninstalling_openshift_ai_self-managed/working-with-certificates_certs#adding-certificates-to-a-custom-ca-bundle_certs[Adding certificates to a custom CA bundle].
endif::[]
ifdef::disconnected[]
Alternatively, you can add certificates to a custom CA bundle, as described in link:{rhoaidocshome}{default-format-url}/installing_and_uninstalling_openshift_ai_self-managed_in_a_disconnected_environment/working-with-certificates_certs#adding-certificates-to-a-custom-ca-bundle_certs[Adding certificates to a custom CA bundle].
endif::[]
endif::[]
ifdef::cloud-service[]
Alternatively, you can add certificates to a custom CA bundle, as described in link:{rhoaidocshome}{default-format-url}/installing_and_uninstalling_openshift_ai_cloud_service/working-with-certificates_certs#adding-certificates-to-a-custom-ca-bundle_certs[Adding certificates to a custom CA bundle].
endif::[]

.Prerequisites

> * You have created a self-signed certificate for (for example, by using opensll) and saved the certificate to a file (for example, example-ca.crt).

Reviewer: ```suggestion
* You have created a self-signed certificate and saved the certificate to a file. For example, you have created a certificate using OpenSSL and saved it to a file named `example-ca.crt`.
```
If this is too wordy, just change "opensll" to "OpenSSL" and delete the extra "for" before the first parenthetical

* You are working in a new installation of {productname-long}. If you upgraded {productname-long}, see link:{odhdocshome}/upgrading-open-data-hub/#adding-a-ca-bundle-after-upgrading_certs[Adding a CA bundle after upgrading].
endif::[]

ifdef::self-managed[]

ifdef::disconnected[] 
* You are working in a new installation of {productname-long}. If you upgraded {productname-long}, see link:{rhoaidocshome}{default-format-url}/upgrading_{url-productname-short}_in_a_disconnected_environment/adding-a-ca-bundle-after-upgrading_upgrade[Adding a CA bundle after upgrading].
endif::[]
ifndef::disconnected[]
* You are working in a new installation of {productname-long}. If you upgraded {productname-long}, see link:{rhoaidocshome}{default-format-url}/upgrading_{url-productname-short}/adding-a-ca-bundle-after-upgrading_upgrade[Adding a CA bundle after upgrading].
endif::[]

endif::[]

ifdef::cloud-service[]
* You are working in a new installation of {productname-long}. If you upgraded {productname-long}, see link:{rhoaidocshome}{default-format-url}/upgrading_{url-productname-short}/adding-a-ca-bundle-after-upgrading_upgrade[Adding a CA bundle after upgrading].
endif::[]

.Procedure

> . Create a config map that includes the root CA certificate used to sign the certificate: 	

Reviewer: ```suggestion
. Create a ConfigMap that includes the root CA certificate used to sign the certificate: 	
```
Changed for consistency with other references to this, but feel free to change if I'm wrong here

+
[source]
----				
oc create configmap custom-ca \
 	--from-file=ca-bundle.crt=</path/to/example-ca.crt> \
 	-n openshift-config
----
+

> `</path/to/example-ca.crt>` is the path to the CA certificate bundle on your local file system. 

Reviewer: Could we make this a callout in the code? Fine if not
